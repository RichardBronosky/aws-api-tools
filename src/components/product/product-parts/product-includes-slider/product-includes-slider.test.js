import React from 'react'
import renderer from 'react-test-renderer'
import ProductIncludesSlider from './index'

describe('ProductIncludesSlider', () => {
  const testProps = {
    items_in_room: [
      {
        sku: '15697064',
        quantity: 1,
        dimensions: "68.5'w x 33.5'd x 40'h",
        image:
          'https://images.rtg-dev.com/cindy-crawford-home-gianna-gray-leather-loveseat_15697064_image-item?cache-id=67eb9144043a65cb6decf007bc83c039',
        generic_name: 'stationary loveseat',
        title: 'Cindy Crawford Home Gianna Gray Leather Loveseat',
      },
      {
        sku: '15797066',
        quantity: 1,
        dimensions: "44.5'w x 33.5'd x 40'h",
        image:
          'https://images.rtg-dev.com/cindy-crawford-home-gianna-gray-leather-glider-recliner_15797066_image-item?cache-id=55382459b39c0650a985a3124a19fbb1',
        generic_name: 'glider recliner',
        title: 'Cindy Crawford Home Gianna Gray Leather Glider Recliner',
      },
      {
        sku: '15197064',
        quantity: 1,
        dimensions: "89'w x 40'd x 40'h",
        image:
          'https://images.rtg-dev.com/cindy-crawford-home-gianna-gray-leather-reclining-sofa_15197064_image-item?cache-id=523e7638d89e935fbe79b88787a2c542',
        generic_name: 'reclining sofa',
        title: 'Cindy Crawford Home Gianna Gray Leather Reclining Sofa',
      },
    ],
    heading: 'test heading',
  }

  it('renders correctly with items in room', () => {
    const tree = renderer.create(<ProductIncludesSlider { ...testProps } />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
