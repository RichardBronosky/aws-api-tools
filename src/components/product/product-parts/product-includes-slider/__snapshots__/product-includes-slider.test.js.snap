// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductIncludesSlider renders correctly with items in room 1`] = `
<div
  className="includes-slider-container small-12"
>
  <section
    className="slider small"
  >
    <div
      className="slider-head"
    >
      <div
        className="slider-heading"
      >
        <h2>
          <span
            className="hide508"
          >
             
          </span>
          test heading
        </h2>
      </div>
      <div
        className="slider-link"
      />
    </div>
  </section>
</div>
`;
